src/components/MovieMain.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+
+import MovieTile from './MovieTile';
+
+class MovieMain extends Component {
```

Comment:
I think this was a good use case for state but also keeping information stored where is relevant. 
I think we could refactor our movie data to live on the app component but this feels good atm and I don't think we *have* to.